modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java

```diff
@@ -337,7 +339,15 @@ else if (objectField.getListTypeDefinitionId() != 0) {
 				return StringPool.BLANK;
 			}
 
-			return listTypeEntry.getName(serviceContext.getLocale());
+			return ListUtil.fromArray(
```

Comment:
Hey @marcelabc , can you check if is it working using Table Template visualization? It can be configured on Style Display property of Collection Display fragment.

Reply:
By the way, I created this technical debt to work on this soon. Brian asked a follow up PR with the sort of this method, but I want to refactor it to avoid this problem of changes in two classes.
[LPS-174042](https://issues.liferay.com/browse/LPS-174042)
Reuse the logic to get the field values on ObjectEntryInfoItemFieldValuesProvider and ObjectEntryRowInfoItemRenderer

Reply:
Hi @guilhermedcamacho,
I did some manual tests regarding the scenarios you showed me earlier in the online meeting. I believe they are working as expected. I printed the screen just to make it registered.
![Screenshot from 2023-02-14 01-09-49](https://user-images.githubusercontent.com/15161924/218637718-3994f820-2a31-471a-9a9b-7cd9bc655d66.png)